mi-scheduler/base/argo-workflows/mi-workflow-template.yaml

```diff
@@ -9,81 +9,34 @@ metadata:
 
 spec:
   serviceAccountName: argo
-  entrypoint: mi-workflow
```

Comment:
why removed?

Reply:
forgot about that one from previously, thx!